.templates/docs.sh.template

```diff
@@ -0,0 +1,101 @@
+{% extends "templates/docs.sh.template" %}
+# Based off NengoDL gpu script. We want to build docs on a machine with FPGAs
+# See https://oncletom.io/2016/travis-ssh-deploy/ for instructions on how the
+# remote server is set up for this script to connect to
+
+{% block install %}
+    # Decrypt private keys in tarball (can't decrypt multiple files currently)
+    # See https://docs.travis-ci.com/user/encrypting-files/
+    exe openssl aes-256-cbc \
+        -K "${encrypted_e86b3ea1b4ca_key:?}" \
+        -iv "${encrypted_e86b3ea1b4ca_iv:?}" \
+        -in .ci/rsa_keys.tar.enc \
+        -out ~/.ssh/rsa_keys.tar -d
+    exe tar -xvf ~/.ssh/rsa_keys.tar -C ~/.ssh
+    exe chmod 600 ~/.ssh/travis-abrgpu_rsa
+    exe chmod 600 ~/.ssh/travis-abrhost_rsa
+
+    # Setup SSH config for abrgpu and abrhost (stored in travis-ci settings)
+    echo -e "$SSH_INFO" >> ~/.ssh/config
+    ssh -o StrictHostKeyChecking=no fpga-ci "echo 'Connected to abrhost'"
+
+    # Setup env on remote server side
+    exe ssh fpga-ci << EOF
+        export STATUS=0  # Status flag for remote
+        $(typeset -f exe)  # Send definition of exe function
+        conda create -y -n travis-ci-"$TRAVIS_JOB_NUMBER" python="$TRAVIS_PYTHON_VERSION"
+        conda activate travis-ci-"$TRAVIS_JOB_NUMBER"
+        exe git clone -b $BRANCH_NAME https://github.com/{{ repo_name }}.git nengo-fpga-"$TRAVIS_JOB_NUMBER"
+
+        # The docs_dir is set in the base and has "../" prepended,
+        # so we enter another dir first
+        cd nengo-fpga-"$TRAVIS_JOB_NUMBER"
+    {{ super() | indent(4) }}
```

Comment:
Do we want to flush the super call, or should we leave it as is?

Reply:
The first line of the `{{ super() }}` block appears to get the indentation of the call, so we want it here I believe, otherwise the single (first) line would be over-indented.

Reply:
Hmmm. It doesn't seem to do it on my system locally... 🤔

Reply:
it does for me... but I guess the user never really looks at the generated script for the most part. So maybe we just indent it properly anyways?

Reply:
I'd be in favour of that. Keeps things consistent.

Reply:
You can set `indent(4, first=True)` to indent the first line (see https://jinja.palletsprojects.com/en/2.10.x/templates/#indent).

Reply:
But yeah the default is `first=False`, which means that it shouldn't be over-indented if you indent the `super` block 🤷‍♂ .

Reply:
who knows, maybe it's a version thing.